hivemind/client/averaging/__init__.py

```diff
@@ -61,7 +60,7 @@ class DecentralizedAverager(mp.Process, averaging_pb2_grpc.DecentralizedAveragin
       towards the (estimated) average by this coefficient. By default, local parameters are set equal to average.
     :param request_timeout: when looking for group, wait for a response from leader for at most this many seconds.
     :note: request_timeout must be smaller than averaging_expiration to avoid potential deadlocks.
-    :param chunk_size_bytes: tensors for AllReduce will be divided into chunks of this size (to improve gRPC throughput)
+    :param part_size_bytes: tensors for AllReduce will be divided into chunks of this size (to improve gRPC throughput)
```

Comment:
Unified glossary:
- peer_fractions = fractions of total averaged_tensors for each peer
- *part* = small tensor sliced from a larger one for All-Reduce
- ~~chunk~~ deprecated, only used in load_state_from peers until we switch to libp2p

Reply:
done that already; the only remaining occurences relate to deprecated split_for_streaming, which will be removed in future

Reply:
OK then, but add it to a corresponding issue so that we don't forget about it

Reply:
added to https://github.com/learning-at-home/hivemind/issues/169